pkgs/development/python-modules/stups-cli-support/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, clickclick
+, dnspython
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "stups-cli-support";
+  version = "1.1.20";
+
+  meta = with lib; {
+    description = "Helper library for all STUPS command line tools.";
+    homepage = "https://github.com/zalando-stups/stups-cli-support";
+    license = licenses.asl20;
+    maintainers = [ maintainers.mschuwalow ];
+  };
+
+  src = builtins.fetchTarball {
```

Since you're pulling from github you'll probably want fetchFromGitHub, but to immediately fix your eval
```suggestion
  src = fetchurl {
```

---

thanks!